crates/egui/src/containers/area.rs

```diff
@@ -607,15 +606,6 @@ impl Prepared {
     }
 }
 
-fn pointer_pressed_on_area(ctx: &Context, layer_id: LayerId) -> bool {
-    if let Some(pointer_pos) = ctx.pointer_interact_pos() {
-        let any_pressed = ctx.input(|i| i.pointer.any_pressed());
-        any_pressed && ctx.layer_id_at(pointer_pos) == Some(layer_id)
-    } else {
-        false
-    }
-}
-
```

Comment:
I had to remove this because it meant you could still focus lower modals even though the click should have been caught by the backdrop. Do you think this code is still necessary @emilk?